.github/workflows/build.yaml

```diff
@@ -2,14 +2,22 @@ name: Build
 on:
   push:
     branches:
-      main
+      - main
+    paths-ignore:
+      - '.gitignore'
```

Comment:
Noting that we are running unit-tests here for other Doc/README changes. I.e it will generate a JAR file on pushes to the main branch. If that's the agreed upon process that we run a full build on all changes, then Im okay with this.

Reply:
That's a good point, I will add ".md" files to the ignore list.